### DISCUSSION TITLE
iOS 16.2 crashes on startup

### DISCUSSION THREAD

**Discussion by CCasusensa** [community/community#110103] [Mobile] [answered]
### Select Topic Area

Question

### Body

My iPhone XS Max Pro on iOS 16.2
In the latest version of the github app it crashes when startup

**Comment by HenSquared** [comment, marked as answer]
@CCasusensa thanks for the information, especially the OS version. We're actively investigating this issue. Sorry for the inconvenience caused. 

We will report back here when we have updates.
